src/pages/code-of-conduct/index.jsx

```diff
@@ -14,62 +14,54 @@ const CodeOfConduct = () => (
+    <h1 className="coc__heading">HackBeanpot 2021 Code of Conduct</h1>
     <div className="coc__body">
       <p>
-        The HackBeanpot team is dedicated to providing a safe, harassment-free
-        experience for everyone, regardless of gender, age, sexual orientation,
-        disability, physical appearance, body size, race, ethnicity,
-        nationality, religion, previous hackathon attendance, or computing
-        experience (or lack of any of the aforementioned). We do not tolerate
-        harassment of hackathon participants in any form. Harassment includes,
-        but is not limited to: offensive comments, sexual language or imagery,
-        deliberate intimidation, stalking, harassing photography or recording,
-        sustained general disruption, inappropriate physical contact, and
-        unwelcome sexual attention.
+        The HackBeanpot team is dedicated to providing a safe, harassment-free 
+        experience for everyone, regardless of gender, age, sexual orientation, 
+        disability, physical appearance, body size, race, ethnicity, 
+        nationality, religion, previous hackathon attendance, or computing 
+        experience (or lack of any of the aforementioned). We do not tolerate 
+        harassment of hackathon participants in any form. Harassment includes, 
+        but is not limited to: offensive comments, sexual language or imagery, 
+        deliberate intimidation, stalking, harassive photography or recording, 
+        sustained general disruption, and unwelcome sexual attention. 
       </p>
       <p>
-        Harassment is not tolerated in any part of the hackathon, including
-        within the venue, as a part of hacks created at the event, on social
-        media, or other online media. Hackathon attendees violating these rules
-        may be warned, disqualified, or expelled from the hackathon at the
-        discretion of the Event Staff.
+        Harassment is not tolerated in any part of the hackathon, including 
+        within our virtual spaces, as a part of hackathon projects created 
+        at the event, or on social media. Hackathon attendees, sponsors, judges, 
+        mentors, organizers, and all other individuals at HackBeanpot 2021 are 
+        subject to this Code of Conduct. Individuals violating these rules 
+        may be warned, disqualified, or expelled from the hackathon at the 
+        discretion of the Event Staff.      
       </p>
       <p>
-        Participants asked to stop any harassing behavior are expected to comply
-        immediately. If a participant engages in harassing behavior during,
-        before, or after the event, the Event Staff may take any action they
-        deem appropriate, including warning the offender, expelling them from
-        the hackathon and removing them from the premises, or excluding them
+        Participants asked to stop any harrassive behavior are expected to comply 
```

Comment:
is this paragraph and line 31-37 the same? What are the differences between the two ?

Reply:
Oh good point... @liu-caro thoughts? I think the last line of the above paragraph can be removed since the paragraph below covers that point.

Reply:
@snwessel talked with Cari, let's remove the last line of the paragraph above. Reflected the updates in the official copy for you to see: https://docs.google.com/document/d/187bJYHsUOjUnx3wnDp1-1o3JHonFoH1VWa53Oasm7nc/edit#

Reply:
Thank you! I'll go ahead and update that.